content/themes/biojs-theme/assets/sass/layout.scss

```diff
@@ -161,7 +161,7 @@ nav {
   min-height: 380px;
   line-height: 0;
   height: auto;
-  color: rgba(255,255,255,.9);
+  // color: rgba(255,255,255,.9);
```

Comment:
@ayusharma As this class `.parallax-container` is a pretty global one used on many several instances on the website, I wouldn't change this css declaration. Instead, you can have a look at the `.hbs` templates of this bug  and try to find out if you might be able to fix the issue by applying one of our [text-coloring helper classes](https://github.com/biojs/biojs.net2/blob/master/content/themes/biojs-theme/assets/sass/main.scss)
This way you should be able to fix this one local text coloring bug without having to check every other instance of the `.parallax-container` class on our website for correct text coloring.